delta/plugins/blazegraph/src/test/resources/routes/responses/indexing-view.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
the vocabulary keys should be sorted properly; is this a test resource issue or there are some keys that are not included in the SortKeys?